framer/Layer.coffee

```diff
@@ -681,17 +691,19 @@ class exports.Layer extends BaseClass
 			# As an optimization, we will only use a loader
 			# if something is explicitly listening to the load event
 
-			if @_domEventManager.listeners(Events.ImageLoaded)?.length > 0 or @_domEventManager.listeners(Events.ImageLoadError).length > 0
-				loader = new Image()
-				loader.name = imageUrl
-				loader.src = imageUrl
+			if @listeners(Events.ImageLoaded, true) or @listeners(Events.ImageLoadError, true) or @listeners(Events.ImageLoadCancelled, true)
```

Comment:
Maybe clean up the image loader after success/error/cancel?

Not sure if it matters here, but `js -> dom -> js` like you do here (js creates dom node, dom node refers to js function) is terrible for garbage collection. So I would actually definitely do a cleanup.